test/language/class/class-to-string.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class to string
+---*/
+class C {}
+assert.sameValue(C.toString(), 'class C {}');
+
+class D { constructor() { 42; } }
+assert.sameValue(D.toString(), 'class D { constructor() { 42; } }');
```

Comment:
Agree, I'm not sure we can get away with this. Suggest removal, unless there is major pushback. I'm not sure impls will agree precisely on toString output (and toString output of class constructors especially seems open to interpretation)